crates/iota-sdk-ffi/src/graphql.rs

```diff
@@ -955,3 +960,85 @@ pub struct Query {
     #[serde(default)]
     pub variables: Option<serde_json::Value>,
 }
+
+impl iota_transaction_builder::ClientMethods for GraphQLClient {
```

Comment:
Can't we just provide a reference to the inner client in the places where this is needed?

Reply:
yes but then we also need to move that logic to another place. Just makes more sense to consider this type a client